src/components/WebexMember/WebexMember.jsx

```diff
@@ -13,22 +13,25 @@ import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
  *
  * @returns {object} JSX of the component
  */
-export default function WebexMember({personID, displayStatus}) {
+export default function WebexMember({personID, displayStatus, isMuted}) {
```

Comment:
Passing `isMuted` as a prop to `WebexMember` does not align with the component system design.
In the component system, each component should be as "self contained" as possible, requiring minimal props. In the design, each component is passed an ID of the resource it presents, and with that ID the component (through 
 a hook that wires in the adapter) fetches the data it needs and listens to changes on that resource to update its UI accordingly.
In this case, instead of passing the mute status as a prop, you need to use the membership hook to get that information about this member.